lib/components/user/monitored-trip/saved-trip-editor.tsx

```diff
@@ -0,0 +1,76 @@
+import { FormattedMessage, useIntl } from 'react-intl'
+import React, { ComponentType } from 'react'
+
+import BackLink from '../back-link'
+import PageTitle from '../../util/page-title'
+import StackedPaneDisplay from '../stacked-pane-display'
+
+interface Props {
+  isCreating: boolean
+  onCancel: () => void
+  panes: Record<string, ComponentType>
+  // TODO: Combine monitored trip types
+  values: Record<string, unknown>
+}
+
+/**
+ * This component handles editing of an existing trip.
+ */
+const SavedTripEditor = (props: Props): JSX.Element => {
+  // The props include Formik props that provide access to the current trip data (stored in props.values)
+  // and to its own blur/change/submit event handlers that automate the state.
+  // We forward the props to each pane so that their individual controls
+  // can be wired to be managed by Formik.
+  const { isCreating, onCancel, panes, values: monitoredTrip } = props
+  const intl = useIntl()
+
+  if (monitoredTrip) {
+    const paneSequence = [
+      {
+        pane: panes.basics,
+        props,
+        title: (
+          <FormattedMessage id="components.SavedTripEditor.tripInformation" />
+        )
+      },
+      {
+        pane: panes.notifications,
+        props,
+        title: (
+          <FormattedMessage id="components.SavedTripEditor.tripNotifications" />
+        )
+      }
+    ]
+
+    const title = isCreating
+      ? intl.formatMessage({ id: 'components.SavedTripEditor.saveNewTrip' })
+      : intl.formatMessage({ id: 'components.SavedTripEditor.editSavedTrip' })
+
+    return (
+      <>
+        <PageTitle title={title} />
+        <BackLink />
+        <StackedPaneDisplay
```

Comment:
Do we sort these too?

Reply:
Not necessarily, because there is a visual order. An invisible component can be anywhere, but I tried to put the title closest to the top.